service/resource_router_test.go

```diff
@@ -0,0 +1,90 @@
+package service
+
+import (
+	"context"
+	"reflect"
+	"testing"
+
+	"github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
+	"github.com/giantswarm/kvm-operator/service/resource/configmapv3"
+	"github.com/giantswarm/kvm-operator/service/resource/servicev2"
+	"github.com/giantswarm/operatorkit/framework"
+)
+
+func Test_NewResourceRouter(t *testing.T) {
```

Comment:
The method being tested is private. The test name implies it is public. It would also be cool to add one more nesting level to the test name to better be able to comprehend where the test might be located at and what it is about. I would like to call this `Test_Service_newResourceRouter`.

Reply:
Done